src/utils/smartClip.ts

```diff
@@ -0,0 +1,51 @@
+import { useEffect, type ClipboardEvent } from "react";
+import { z } from "zod";
+import { api } from "./api";
+
+export const useSmartClip = ({
```

Comment:
Same for this

Reply:
I've deleted this file. The api hook can't be called outside of React components anyway.